cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -10,6 +10,7 @@ Usage: cardano-cli
                      | Legacy commands
                      | byron
                      | ping
+                     | Debug commands
```

Comment:
Why `Legacy commands` and `Debug commands` are upper case? This is a bit confusing to me.

Reply:
Updated.